core/src/main/java/io/aiven/klaw/service/BaseOverviewService.java

```diff
@@ -260,7 +261,16 @@ protected List<AclOverviewInfo> applyFiltersAclsForSOT(
         // Claim Acl is available to teams that are unable to delete the Acl as they do not own the
         // ACL.
         // Will need to add an extra check when an acl claim is opened.
-        mp.setShowClaimAcl(!mp.isShowDeleteAcl());
+        mp.setShowClaimAcl(
+            !mp.isShowDeleteAcl()
+                && !manageDatabase
+                    .getHandleDbRequests()
+                    .existsSpecificAclRequest(
+                        mp.getTopicname(),
+                        RequestStatus.CREATED.value,
+                        mp.getEnvironment(),
+                        tenantId,
+                        Integer.valueOf(mp.getReq_no())));
```

Comment:
```suggestion
                        Integer.parseInt(mp.getReq_no())));
```

parseInt is better, and we can remove that boxing